toolkit/docs/building/building.md

```diff
@@ -292,10 +294,19 @@ While `tdnf` uses a list of repo files:
 REPO_LIST ?=
 ```
 
-The `REPO_LIST` variable supports multiple repo files, and they are prioritized in the order they appear in the list.
+The `REPO_LIST` variable supports multiple, space-separated repo files, and they are prioritized in the order they appear in the list.
 The CBL-Mariner base and update repos are implicitly provided and an optional preview repo is available by setting `USE_PREVIEW_REPO=y`.
 To disable the update repo set `USE_UPDATE_REPO=n`. If `DISABLE_UPSTREAM_REPOS=y` is set, any repo that is accessed through the network is disabled.
 
+The tooling will use the first package it can find by looking through the repositories in the following order:
+
+1. the worker chroot environment;
+2. already built local RPMs in `out/RPMS/`;
+3. the core CBL-Mariner upstream **preview** repository if `USE_PREVIEW_REPO` is set to `y`;
+4. the core CBL-Mariner upstream **update** repository if `USE_UPDATE_REPO` is **not** set to `n`;
+5. the core CBL-Mariner upstream **base** repository;
```

Comment:
Can someone please double-check this order? I couldn't figure out what's happening if we're using both the preview and update repository.